commands/build_test.go

```diff
@@ -39,19 +39,19 @@ func Test_parseBuildArgs_ValidParts(t *testing.T) {
 }
 
 func Test_parseBuildArgs_NoSeparator(t *testing.T) {
-	_, err := parseBuildArgs([]string{"kv"})
+	_, err := parseMap([]string{"kv"}, "build-arg")
 
-	want := "each build-arg must take the form key=value"
+	want := "label format is not correct, needs key=value"
 	if err != nil && err.Error() != want {
 		t.Errorf("Expected an error due to missing seperator")
 		t.Fail()
 	}
 }
 
 func Test_parseBuildArgs_EmptyKey(t *testing.T) {
-	_, err := parseBuildArgs([]string{"=v"})
+	_, err := parseMap([]string{"=v"}, "build-arg")
```

Comment:
Are there existing tests covering `parseMap` or are these the only ones? If they are the only ones then we should rename the test file too.

Reply:
these the only ones. How about creating a `utils.go` under `commands` package for  `parseMap`and other common utils.  We can separately have  test cases for `utils`